### DISCUSSION TITLE
Is it (still) possible to bypass the SEB with a VM?

### DISCUSSION THREAD

**Discussion by sbrunold** [SafeExamBrowser/seb-win-refactoring#901] [Q&A]
There are videos online that show how to use the SEB via a virtual machine and thus bypass a secure exam environment (https://www.youtube.com/watch?v=U8rnA3ZSiPs) by running the SEB via the virtual machine, but still having access to the Internet, for example, on the normal desktop.
Is this still possible? Can you recognize this as a teacher or has this "bypass" been prevented?
Thanks for the clarification.

**Comment by dbuechel** [comment]
No, the video is showcasing a bypass with the legacy version (2.x). This particular bypass should not work anymore with the latest release versions of SEB 3.x.
